setup.cfg

```
@@ -46,7 +46,7 @@ install_requires =
     popylar==0.2.*
     imageio>=2.8.0,<3.0.0
     templateflow==0.6.2
-    pybids>=0.11.0,<0.13.0
+    pybids==0.9.3
```

Can we do this?
```suggestion
    pybids>=0.9.3, <0.13.0
```
to be a little bit more flexible and avoid potential dependency conflicts?

---

Yes I am just forcing this version to run the CLI tests on it for now (: . I will switch it to that once they pass